edx_toggles/tests/test_waffle.py

```diff
@@ -29,6 +29,17 @@ def test_constructor(self):
         self.assertEqual("module1", waffle.module_name)
         self.assertEqual(1, len(NaiveWaffle.get_instances()))
 
+    def test_toggle_methods(self):
+        waffle = NaiveWaffle("namespaced.name", "module1")
+        # test is_enabled method
+        self.assertEqual(True, waffle.is_enabled())
+        #test is_disabled method
```

Comment:
Nit: Please add this space to the remaining comments.
```suggestion
        # test is_disabled method
```